### PR TITLE
Thilina  test

### PR DESCRIPTION
**Pull request by ThilineTissera**

### CODE REVIEW

File: src/main/java/com/techbirdssolutions/pos/PosApplication.java [hunk 1]
```diff
@@ -3,11 +3,19 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
+import java.sql.SQLOutput;
```
Comment (sachinkumaraliyanage) [inline src/main/java/com/techbirdssolutions/pos/PosApplication.java]: remove this unuse import

File: src/main/java/com/techbirdssolutions/pos/PosApplication.java [hunk 2]
```diff
@@ -3,11 +3,19 @@
...
+
 @SpringBootApplication
-public class PosApplication {
+public class PosApplication
```
Comment (sachinkumaraliyanage) [inline src/main/java/com/techbirdssolutions/pos/PosApplication.java]: follow the standard ad class name an { in same line

**Review by sachinkumaraliyanage — CHANGES_REQUESTED** [review]

File: src/main/java/com/techbirdssolutions/pos/PosApplication.java (hunk 1 above)
Comment (ThilineTissera) [reply src/main/java/com/techbirdssolutions/pos/PosApplication.java]: resolved

File: src/main/java/com/techbirdssolutions/pos/PosApplication.java (hunk 2 above)
Comment (ThilineTissera) [reply src/main/java/com/techbirdssolutions/pos/PosApplication.java]: resolved

**Pull request closed without merging**